hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -82,7 +113,7 @@ const COLUMNS = [
     key: 'replicas',
     render: (replicas: IContainerReplica[]) => (
       <div>
-        {replicas.map(replica => {
+        {replicas && replicas.map(replica => {
```

Comment:
Can you help me understand why do we need this?

Reply:
It is standard practice to avoid undefined  and empty array mapping. Just to make sure we have valid data before map.